index.html

```diff
@@ -1,42 +1,32 @@
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <title>Jasmine Spec Runner</title>
-  <link rel="shortcut icon" type="image/png" href=".grunt/grunt-contrib-jasmine/jasmine_favicon.png">
-
-  <link rel="stylesheet" type="text/css" href=".grunt/grunt-contrib-jasmine/jasmine.css">
-
-
-</head>
-<body>
-
-  
-  <script src=".grunt/grunt-contrib-jasmine/jasmine.js"></script>
-  
-  <script src=".grunt/grunt-contrib-jasmine/jasmine-html.js"></script>
-  
-  <script src=".grunt/grunt-contrib-jasmine/json2.js"></script>
-  
-  <script src=".grunt/grunt-contrib-jasmine/boot.js"></script>
-
-  <script src="node_modules/babel-polyfill/dist/polyfill.js"></script>
-
-  <script src="node_modules/jasmine-promises/dist/jasmine-promises.js"></script>
-  
-
-  <script src="main.js"></script>
-
-  <script>
-  jasmine.DEFAULT_TIMEOUT_INTERVAL = 15000;
-  </script>
-
-  
-  <!--/*<script src="spec/fhSyncCreateSpec.js"></script>*/-->
-  <script src="spec/syncSpec.js"></script>
-  
-  <script src=".grunt/grunt-contrib-jasmine/reporter.js"></script>
-  
-
-</body>
+  <head>
```

Comment:
Yes they were, although they must have been squashed in here by accident, just a clean up of the file